packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.tsx

```diff
@@ -171,10 +190,12 @@ const CreateRancher: React.FC<CreateRancherProps> = ({
           {plans?.map((v) => (
```

Comment:
I think we prefer to avoid "letter variable"
```suggestion
          {plans?.map((plan) => (
```